go.sum

```diff
@@ -545,6 +545,8 @@ github.com/stretchr/testify v1.2.2/go.mod h1:a8OnRcib4nhh0OaRAV+Yts87kKdq0PP7pXf
 github.com/stretchr/testify v1.3.0/go.mod h1:M5WIy9Dh21IEIfnGCwXGc5bZfKNJtfHm1UVUgZn+9EI=
 github.com/stretchr/testify v1.4.0 h1:2E4SXV/wtOkTonXsotYi4li6zVWxYlZuYNCXe9XRJyk=
```

Comment:
@lingxiankong looks like old dependencies still exist, please run go mod tidy to cleanup the same.

Reply:
weird, I scrolled up a little bit, found there are many modules that have different versions, did you happen to know the reason?

Reply:
ya, strange , I thought go mod tidy should do required cleanup of the older versions?
One reason may be older versions are some indirect dependency of some package, but not sure

Reply:
it is only a hashsum, not a direct dependency. if you remove go.sum and run `go mod tidy` again, it should cleanup old dep hashes.

Reply:
@kayrus Do you mean we need to remove go.sum file every time before modifying go.mod in order to remove those hashsum? Does the hashsum bring any effect to the build process?

Reply:
> Does the hashsum bring any effect to the build process?
no
> Do you mean we need to remove go.sum file every time before modifying go.mod in order to remove those hashsum?
I'd recommend to ignore this next time